Test/prover/batch-solver.bpl

```diff
@@ -0,0 +1,8 @@
+// RUN: %boogie /trace /proverOpt:BATCH_MODE=true "%s" > "%t"
```

Comment:
I might be good to convert some number of existing tests to use batch mode, say 20 of them.

Reply:
I'm currently working on getting all except for a select subset running in batch mode, as it turns out. Hence the failing test runs at the moment while I debug the script. :)